fuse_publishers/test/test_pose_2d_publisher.cpp

```diff
@@ -31,66 +31,76 @@
  *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  *  POSSIBILITY OF SUCH DAMAGE.
  */
+
+// Workaround ros2/geometry2#242
```

Comment:
Same comment about comment being in wrong spot, or workaround not actually being needed.

Reply:
I ended up reordering them just in case
https://github.com/locusrobotics/fuse/pull/305/commits/166b39a5f8a06ff4e648aef37cd3bea47ea17114